serve/destination.go

```diff
@@ -103,7 +103,7 @@ func newCmdDestinationServe(destination *destinationServe) *cobra.Command {
 				err = sentry.Init(sentry.ClientOptions{
 					Dsn:              destination.sentryDSN,
 					Debug:            false,
-					AttachStacktrace: true,
+					AttachStacktrace: false,
```

Comment:
This makes sense after reading the docs:
https://docs.sentry.io/platforms/unity/configuration/options/#attach-stacktrace

![image](https://user-images.githubusercontent.com/26760571/198878733-1a736933-3ac9-4dcb-b24a-68d29bbfb5ea.png)

Reply:
Stacktrace in Go with sentry don't work well at all as it's trying to do some smart things like compiling in the source code which is unnecessary and makes also the even on sentry confusing.